### PR TITLE
Feature isa : adding mw2isa as a tool

### PR DESCRIPTION
**Pull request by proccaserra**
adding mw2isa as a tool

### CODE REVIEW

File: config/job_conf.xml [hunk 1]
```diff
@@ -223,14 +223,14 @@
             <param id="max_pod_retrials">3</param>
             <param id="docker_enabled">true</param>
         </destination>
-        <destination id="container-isajson-validator" runner="k8s">
+        <destination id="container-mw2isa" runner="k8s">
```
Comment (pcm32) [inline config/job_conf.xml]: Question @proccaserra : why are you overwritting container-isajson-validator with container-mwisa?

Comment (proccaserra) [inline config/job_conf.xml]: good question: a bad case of copy-paste?

File: tools/phenomenal/isatools/mw2isa.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="isatab_validator" name="isatab_validator" version="1.0">
+  <description>Produce a validation report on an ISA-Tab ZIP archive</description>
```
Comment (pcm32) [inline tools/phenomenal/isatools/mw2isa.xml]: Thanks @proccaserra for all the changes. Could you please let me know if description of this tool makes sense with what is available on the `<help>...</help>` part of this XML? Seems to me that command, inputs, description and help conflict in some ways. For instance, the "mw_study_id" input is not used in the command.

Comment (proccaserra) [inline tools/phenomenal/isatools/mw2isa.xml]: @pcm32 was in fact fixing this pb now, will push later in the day when I get the expected output

**Pull request merged**